_includes/integrations/expressjs-quick-start.md

```diff
@@ -134,15 +123,15 @@ This procedure requires:
     * To start the development server, navigate to the `my_{{site.prodnamecode}}_app` directory and run:
 
         ```sh
-        $ npm start
+        npm start
         ```
 
-    * Visit http://localhost:3000 to view the application.
+    * Visit <http://localhost:3000> to view the application.
```

Comment:
Is this correct? As this looks like it should be a link and that isn't how links are done in Markdown.